www/utils/packages/docblock-generator/src/classes/kinds/oas.ts

```diff
@@ -632,11 +632,8 @@ class OasKindGenerator extends FunctionKindGenerator {
     normalized: string
   } {
     const filePath = node.getSourceFile().fileName
-    const oasPath = (
-      filePath.includes("/api-v2/")
-        ? filePath.substring(filePath.indexOf("/api-v2/"))
-        : filePath.substring(filePath.indexOf("/api/"))
-    )
+    const oasPath = filePath
+      .substring(filePath.indexOf("/api/"))
       .replace(/^\/api(-v2)?\//, "")
```

Comment:
suggestion: we can probably remove the v2 notion entirely?

Reply:
good catch, removed!